### PR TITLE
added resetPassword and first_time_setPassword functions and screens

### PR DESCRIPTION
**Pull request by ericfu666**
For firebase reset password service, there's like a 150 times/day limit. Avoid trying too much

### CODE REVIEW

File: src/pages/ForgetPassword.jsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+              }}
+            >
+              Back
+            </button>
+          </form>
+
+          <div className="text-sm mt-32">
+            <div className="flex justify-start gap-4">
+              <EmailIcon fontSize="small" />
+              Don&apos;t have email registered yet with RefugeeOne?{" "}
```
Comment (nehakumari2307) [inline src/pages/ForgetPassword.jsx:92]: Remove the white space at the end

File: src/pages/SetPassword.jsx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+  return (
+    <div className="flex flex-col justify-center items-center">
+      <div className="card max-w-md">
+        <div className="card-body">
+          <p className="mb-5 text-3xl">Welcome!</p>
+          <p className="mb-1 text-xl">Please enter your email</p>
+          <p className="mb-1 text-sm">
+            * Please make sure your email has been registered by the RefugeeOne team. Once
+            you enter a registed email, we will send you a link to set your account
```
Comment (nehakumari2307) [inline src/pages/SetPassword.jsx:53]: Can we add a info icon before all the notes

**Review by nehakumari2307 — APPROVED** [review]

File: src/pages/ForgetPassword.jsx (hunk 1 above)
Comment (ericfu666) [reply src/pages/ForgetPassword.jsx]: fixed

File: src/pages/SetPassword.jsx (hunk 2 above)
Comment (ericfu666) [reply src/pages/SetPassword.jsx:55]: fixed

**Review by makersmelx — APPROVED** [review]

**Pull request merged**